docs/layouts/shortcodes/generated/expert_rest_section.html

```diff
@@ -20,6 +20,18 @@
             <td>Long</td>
             <td>The time in ms that the client waits for the leader address, e.g., Dispatcher or WebMonitorEndpoint</td>
         </tr>
+        <tr>
+            <td><h5>rest.cache.checkpoint-statistics.size</h5></td>
+            <td style="word-wrap: break-word;">1000</td>
+            <td>Integer</td>
+            <td>Maximum number of entries in the checkpoint statistics cache.</td>
+        </tr>
+        <tr>
+            <td><h5>rest.cache.checkpoint-statistics.timeout</h5></td>
+            <td style="word-wrap: break-word;">3 s</td>
+            <td>Duration</td>
+            <td>Duration from write after which cached checkpoints statistics are cleaned up. It can be specified using notation: "500 ms", "1 s".</td>
+        </tr>
```

Comment:
Are these configs backwards compatible? Seems like `rest.cache.checkpoint-statistics.timeout` is a new config split for `web.refresh-interval`?

Reply:
Ok, made it such that it will fallback to `web.refresh-interval`